### ISSUE TITLE
Panic on EstimateTransactionCost 

### ISSUE THREAD

**Issue by peterlimg** [0chain/0chain#1073]
I got panic on `EstimateTransactionCost` , the `mc.GetLatestFinalizedBlock.ClientState` could be nil, which will then cause the panic.

```
miner_1       | panic: runtime error: invalid memory address or nil pointer dereference
miner_1       | [signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x108d710]
miner_1       |
miner_1       | goroutine 187398 [running]:
miner_1       | 0chain.net/chaincore/chain.CreateTxnMPT(0x0, 0x0, 0x0, 0x0)
miner_1       |         /0chain/go/0chain.net/chaincore/chain/state.go:615 +0x50
miner_1       | 0chain.net/chaincore/chain.(*Chain).EstimateTransactionCost(0xc00044ef00, 0x16f3ef0, 0xc0008d2480, 0xc001d3ed20, 0x0, 0x0, 0xc000774000, 0x0, 0x0, 0x0)
miner_1       |         /0chain/go/0chain.net/chaincore/chain/state.go:160 +0x67
miner_1       | 0chain.net/miner.(*Chain).VerifyBlock(0x1c30400, 0x16f3ef0, 0xc0008d2480, 0xc001d3ed20, 0x0, 0x0, 0x0)
miner_1       |         /0chain/go/0chain.net/miner/protocol_block.go:290 +0x12db
miner_1       | 0chain.net/miner.(*Chain).VerifyRoundBlock(0x1c30400, 0x16f3ef0, 0xc0008d2480, 0x16ff060, 0xc000f5f3b0, 0xc001d3ed20, 0x0, 0x0, 0x0)
miner_1       |         /0chain/go/0chain.net/miner/protocol_round.go:1022 +0x9d4
miner_1       | 0chain.net/miner.(*Chain).CollectBlocksForVerification.func1(0x16f3ef0, 0xc0008d2480, 0xc000f5f3b0, 0xc001d3ed20, 0x0)
miner_1       |         /0chain/go/0chain.net/miner/protocol_round.go:840 +0x73e
miner_1       | 0chain.net/miner.(*Chain).CollectBlocksForVerification(0x1c30400, 0x16f3ef0, 0xc0008d2480, 0xc000f5f3b0)
miner_1       |         /0chain/go/0chain.net/miner/protocol_round.go:972 +0x832
miner_1       | created by 0chain.net/miner.(*Chain).StartVerification
```

Actually it is the lfb.ClientState is nil, the lfb should always be none nil, but lfb.ClientState could be nil when a miner get block from sharder, check the code here:
https://github.com/0chain/0chain/blob/90db88e30ca83f772624a87dd05befd6d8d2feb5/code/go/0chain.net/miner/protocol_round.go#L1756.

**Comment by ma2b0043** [comment]
@peterlimg when can we expect to close this issue?

**Comment by peterlimg** [comment]
Closed by #1075
